tutorials/scripting/c_sharp/c_sharp_basics.rst

```diff
@@ -29,9 +18,8 @@ it is implemented with .NET 6.0.
 
 .. attention::
 
-    Projects written in C# using Godot 4.x currently cannot be exported to
-    Android, iOS and web platforms. To use C# on those platforms, use Godot 3
-    instead.
+    Projects written in C# using Godot 4.x currently cannot be exported to iOS
```

Comment:
The experimental Android export only supports architectures arm64 and x64. Should this mentioned here?

Reply:
I would probably add it to the list of features page and then reference it from here.